recipes/distributed/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "1.10.1" %}
+
+package:
+  name: distributed
+  version: "{{ version }}"
+
+source:
+  fn: distributed-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/distributed/distributed-{{ version }}.tar.gz
+  md5: 98c6ec894386a93ffd16718d04b79671
+
+build:
+  number: 0
+  entry_points:
+    - dcenter=distributed.cli.dcenter:go
+    - dscheduler=distributed.cli.dscheduler:go
+    - dcluster=distributed.cli.dcluster:start
+    - dworker=distributed.cli.dworker:go
+  script: python setup.py install
```

Comment:
Please change to `python setup.py install --single-version-externally-managed --record=record.txt`.